src/App.tsx

```diff
@@ -46,7 +46,7 @@ function App(): ReactElement {
   const [isInitialDatasetLoaded, setIsInitialDatasetLoaded] = useState(false);
   const [datasetOpen, setDatasetOpen] = useState(false);
 
-  const [colorRampData] = useState(DEFAULT_COLOR_RAMPS);
+  const colorRampData = DEFAULT_COLOR_RAMPS;
```

Comment:
I like that you kept this assignment, even though it's just an alias to an already available value - should make it easy to change back to state later.